### ISSUE TITLE
How is the segmentation mask given to the model for multi label segmentation mask

### ISSUE THREAD

**Issue by nobleaustine** [SuperMedIntel/MedSegDiff#214]
If the segmentation mask has just two class background and cancer we can give it as mask with 1 and 0s If there are multiple classes, how are they given, we can't do diffusion directly on the segmentation mask right, is onehot encoding done or any  other strategy

**Comment by nobleaustine** [comment]
res = torch.where(mask > 0, 1, 0)  can we run without this code in the training_losses_segmentation

**Comment by nobleaustine** [comment]
For dataset like BTCV with multiple labels how is the training done

**Comment by 15629932373** [comment]
> 对于像 BTCV 这样具有多个标签的数据集，训练是如何完成的

Pls have you solved this issue at the moment
